res/openvpn-gui-res-cs.rc

```diff
@@ -38,8 +38,7 @@ END
 
 /* Auth Username/Password Dialog */
 ID_DLG_AUTH DIALOG 6, 18, 160, 95
-STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER | DS_SETFOREGROUND
-CAPTION "OpenVPN - Ověření uživatele"
```

Comment:
The caption line may be left as is --  it will overwritten in INITDIALOG as you have above.

Reply:
pardon ? why to leave caption line if it will be overwritten ?